pallets/schemas/src/lib.rs

```diff
@@ -269,8 +270,8 @@ pub mod pallet {
 		/// Retrieve a schema by id
 		pub fn get_schema_by_id(schema_id: SchemaId) -> Option<SchemaResponse> {
 			if let Some(schema) = Self::get_schema(schema_id) {
-				// this should get a BoundedVec out
-				let model_vec = schema.model.into_inner();
+				let model_vec: Vec<u8> = schema.model.into_inner();
+				Self::ensure_valid_model(&schema.model_type, &model_vec).ok()?;
```

Comment:
Not sure why we are doing `ensure_valid_model` here. Only valid models can be stored.

Reply:
In theory, yes, but we don't _know_ that... especially since the type of the schema is a vector.

Reply:
True, but this is an on-chain call right?. Everytime we get a schema, I don't think we want to be testing that it is a valid schema.